src/StatusAlert/index.jsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+import styles from './StatusAlert.scss';
+
+class StatusAlert extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.close = this.close.bind(this);
+    this.handleKeyDown = this.handleKeyDown.bind(this);
+    this.renderDialog = this.renderDialog.bind(this);
+
+    this.state = {
+      open: props.open,
+    };
+  }
+
+  componentDidMount() {
+    if (this.xButton) {
+      this.xButton.focus();
+    }
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.open !== this.props.open) {
+      this.setState({ open: nextProps.open });
+    }
+  }
+
+  componentDidUpdate(prevState) {
+    if (this.state.open && !prevState.open) {
+      this.xButton.focus();
+    }
+  }
+
+  close() {
+    this.setState({ open: false });
+    this.props.onClose();
+  }
+
+  handleKeyDown(e) {
+    if (e.key === 'Enter' || e.key === 'Escape') {
+      e.preventDefault();
+      this.close();
+    }
+  }
+
+  renderDialog() {
+    const { dialog } = this.props;
+
+    return (
+      <div className="alert-dialog">
+        { dialog }
+      </div>
+    );
+  }
+
+  renderDismissible() {
+    const { dismissible } = this.props;
+    let dismissButton = (<div />);
```

Comment:
I can definitely give this a go -- it didn't cross my mind that null was a valid thing that could be returned as part of a render.

Reply:
@arizzitano I updated this to be null instead of the empty div.  When I made my first attempt to turn it into a ternary operator it didn't render happily with the <button> spanning over multiple lines.  I will revisit this though when I test out the options to switch to the paragon Button from the above comment.